spring-boot-project/spring-boot-docs/src/docs/asciidoc/using/devtools.adoc

```diff
@@ -287,7 +287,6 @@ You can configure global devtools settings by adding any of the following files
 
 . `spring-boot-devtools.properties`
 . `spring-boot-devtools.yaml`
-. `spring-boot-devtools.yml`
```

Comment:
This should not change as we still look for `.yml` files. The current ordering suggests that `.yaml` is preferred so it is fine as-is.